sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -58,7 +58,7 @@ public FormRecognizerClient(Uri endpoint, AzureKeyCredential credential, FormRec
 
             Diagnostics = new ClientDiagnostics(options);
             var pipeline = HttpPipelineBuilder.Build(options, new AzureKeyCredentialPolicy(credential, Constants.AuthorizationHeader));
-            ServiceClient = new FormRecognizerRestClient(Diagnostics, pipeline, endpoint.AbsoluteUri);
+            ServiceClient = new FormRecognizerRestClient(Diagnostics, pipeline, endpoint.AbsoluteUri, FormRecognizerClientOptions.GetVersionString(options.Version));
```

Comment:
should the apiVersion option be documented in FormRecognizerClientOptions?

Reply:
In python we do validation to ensure the apiVersion passed is a valid one. Does that make sense in .NET?

Reply:
Yes! We actually have an enum with the possible values, and `FormRecognizerClientOptions` only [accepts as a value](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClientOptions.cs#L23) that enum.
In the weird case that something happened and a non-supported version was passed, we have a [validation](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClientOptions.cs#L52) too.

Reply:
nice! sorry I missed that